package.json

```diff
@@ -27,6 +27,8 @@
     "chai-as-promised": "6.0.0",
     "chai-immutable": "1.6.0",
     "copyfiles": "^1.2.0",
+    "coveralls": "^2.13.0",
+    "istanbul": "^0.4.5",
```

Comment:
Recommend not auto-upgrading (`"0.4.5"` instead of `"^0.4.5"`) to avoid things breaking randomly one day.